packages/node/src/schematics/library/files/lib/src/lib/__fileName__.ts__tmpl__

```diff
@@ -0,0 +1,3 @@
+export function add(a: number, b: number): number {
```

Comment:
Perhaps make this function named after the lib and return a string of the lib name.

For example `ng g @nrwl/node:lib cats` will generate:

```ts
export function getCats(): string {
  return 'get-cats';
}
```